openprescribing/frontend/tests/functional/test_measures.py

```diff
@@ -75,6 +75,12 @@ def test_all_england(self):
             'Compare all CCGs in England on this measure',
             '/measure/core_0/'
         )
+        self._verify_link(
+            panel_element,
+            '.inner li:nth-child(2)',
+            'Break the overall score down into individual presentations',
+            '/measure/core_0/all-england/'
+        )
```

Comment:
We should have a test case for the new page.  The code that generates these new pages is quite hard to reason about (which links appear on which pages?) so these tests act as a kind of spec.  It's likely we'll need to touch this code as part of the org refactor, so a new test case would help prevent regressions there.